test/topics/health.test.js

```diff
@@ -29,10 +29,11 @@ describe('Health check', () => {
 		before(async () => {
 			busApi = await bus.start({ config });
 			await busApi.purgeDlqBySubcriptionId('assess');
+			await bus.stop();
 		});
 
-		after(async () => {
-			await bus.stop();
+		beforeEach(async () => {
+			busApi = await bus.start({ config });
```

Comment:
Aren't we missing the `after`?
```
after(async () => {
			await bus.stop();
});
```

Reply:
https://github.com/guidesmiths/systemic-azure-bus/pull/54/commits/f75a47bbc4b4c1b0bdcbba9ef1fc2fa7937d37f2